client/src/resources/lang/en_us.json

```diff
@@ -569,7 +569,7 @@
     "wiki.article.standard.outlineList.title":"Outline List",
     "wiki.article.standard.outlineList.text": "The outline list defines what roles are in the game. At the start of a game, each player gets assigned to a role outline. Then, their role is generated randomly from that role outline.",
     "wiki.article.standard.chatGroup.title": "Chat Group",
-    "wiki.article.standard.chatGroup.text": "Messages are sent to chat groups. The chat groups are:\n - All (no icon)\n - Dead 💀\n - Mafia 🌹\n - Vampire 🧛‍♂️\n - Haunt 👻\n - Jail ⛓️\n - Interview 📝\n\n Players can be in multiple chat groups at once, or even read chat groups they are not in.",
+    "wiki.article.standard.chatGroup.text": "Messages are sent to chat groups. The chat groups are:\n - All (no icon)\n - Dead 💀\n - Mafia 🌹\n - Cult 🪬\n - Haunt 👻\n - Jail ⛓️\n - Interview 📝\n\n Players can be in multiple chat groups at once, or even read chat groups they are not in.",
```

Comment:
idk what this emoji is but my computer doesn't have it

Reply:
You should fix that. Its a really cool emoji

Reply:
Im fixing it dont touch

Reply:
U did not fix it